.changeset/honest-poets-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-bugsnag': patch
```

Comment:
Should we make this minor instead. Would the new annotation option count as a new feature? I think it would

Reply:
I was thinking about it too, but in the end it seemed as an optimisation and fixing non working stuff rather than new feature so I went with the patch